client/my-sites/plan-features/index.jsx

```diff
@@ -463,6 +467,8 @@ export default connect(
 			const showMonthly = ! isMonthly( plan );
 			const available = isInSignup ? true : canUpgradeToPlan( plan );
 			const relatedMonthlyPlan = showMonthly ? getPlanBySlug( state, getMonthlyPlanByYearly( plan ) ) : null;
+			const isCurrentlyPremium = isCurrentSitePlan( state, selectedSiteId, plansList[ PLAN_PREMIUM ].getProductId() );
```

Comment:
I think we can move this out from the `map` function as it doesn't use the `plan` variable. You'll also need to move the `selectedSiteId` definition.